docs/guides/social-signin/data-mapping.mdx

```diff
@@ -0,0 +1,161 @@
+---
+id: data-mapping
+title: Data Mapping with Jsonnet
+---
+
+Different social sign-in providers will send different data in their payloads. While you might expect to get some basic information
```

Comment:
Have you changed anything in this document or is it just copied over?

Reply:
It's been moved with minor updates so have a quick look please. Maybe check if the external claims list is up-to-date? Thanks.